tests/Integration/Schema/Directives/BuilderDirectiveTest.php

```diff
@@ -79,12 +147,13 @@ public function testCallsCustomBuilderMethodOnFieldWithoutValue(): void
     }
 
     /**
-     * @param  \Illuminate\Database\Query\Builder|\Illuminate\Database\Eloquent\Builder  $builder
+     * @param  \Illuminate\Database\Query\Builder|Builder  $builder
+     * @param  int  $value
      *
-     * @return \Illuminate\Database\Query\Builder|\Illuminate\Database\Eloquent\Builder
+     * @return \Illuminate\Database\Query\Builder|Builder
      */
-    public static function limit(object $builder, int $value = 2): object
+    public static function limit(object $builder, $value): object
     {
-        return $builder->limit($value);
+        return $builder->limit($value ?: 2);
```

Comment:
Why not a real default?

Reply:
Well some arguments are added after it so a default would be useless in `BuilderDirective.php` we have the following now:

```php
    public function handleFieldBuilder(object $builder, $root, array $args, GraphQLContext $context, ResolveInfo $resolveInfo): object
    {
        $resolver = $this->resolver();

        if ($this->directiveHasArgument('value')) {
            return $resolver(
                $builder,
                $this->directiveArgValue('value'),
                $root,
                $args,
                $context,
                $resolveInfo
            );
        }

        return $resolver($builder, null, $root, $args, $context, $resolveInfo);
    }
```

So null is the default, unless we decide to move the `$value` at the end but that means the users need to always define 6 variables in their custom methods.